pallets/committee-management/src/impls.rs

```diff
@@ -138,12 +138,13 @@ fn calculate_adjusted_session_points(
     blocks_to_produce_per_session: u32,
     blocks_created: u32,
     total_possible_reward: u32,
+    lenient_threshold: Perquintill,
 ) -> u32 {
     let performance =
         Perquintill::from_rational(blocks_created as u64, blocks_to_produce_per_session as u64);
 
-    // when produced more than 90% expected blocks, get 100% possible reward for session
-    if performance >= LENIENT_THRESHOLD {
```

Comment:
Please raise a task to update wallet https://github.com/Cardinal-Cryptography/apps/blob/alephzero/packages/page-staking/src/Performance/CurrentList.tsx#L48 code to use `committee_management.get_lenient_threshold_percentage` instead of hardcoded value